cmd/minishift/cmd/start_preflight.go

```diff
@@ -49,24 +49,18 @@ const (
 )
 
 // preflightChecksBeforeStartingHost is executed before the startHost function.
-func preflightChecksBeforeStartingHost() {
+func preflightChecksBeforeStartingHost(proxyConfig *util.ProxyConfig) {
 	driverErrorMessage := "See the 'Setting Up the Virtualization Environment' topic (https://docs.openshift.org/latest/minishift/getting-started/setting-up-virtualization-environment.html) for more information"
 	prerequisiteErrorMessage := "See the 'Installing Prerequisites for Minishift' topic (https://docs.openshift.org/latest/minishift/getting-started/installing.html#install-prerequisites) for more information"
 
-	proxy := os.Getenv("HTTPS_PROXY")
-
-	// In case we're not running with -v5, we want to obscure the proxy but
-	// still tell the user whether or not a proxy is being used. This is for
-	// security reasons as proxy URLs might contain credentials thay should not
+	// We want to obscure the proxy but still tell the user whether or not a proxy is being used.
+	// This is for security reasons as proxy URLs might contain credentials they should not
 	// always be shown.
-	if !glog.V(5) {
-		if os.Getenv("HTTPS_PROXY") == "" {
-			proxy = "No"
-		} else {
-			proxy = "Yes"
-		}
+	if proxyConfig.HttpsProxy() != "" || proxyConfig.HttpProxy() != "" {
+		fmt.Println("-- Using proxy for the setup")
```

Comment:
I am having second thoughts about why we want to keep this preflight check as it is not doing any check.

Reply:
Agree :+1: 
This message is important when we use proxy flag in start.

Reply:
I think i got it, you want to put it somewhere else, as this is not a check and just letting the user know if proxy is used or not..

Reply:
@anjannath Right, there is no reason it should be in preflight check.